web/src/utils/exporter.js

```diff
@@ -0,0 +1,74 @@
+
+export function svg2png(svgElement, options = {}) {
+  const findStyles = options.styles !== false;
+  const returnSvg = options.return === 'svg';
+
+  // based on http://bl.ocks.org/biovisualize/8187844
+  const copy = svgElement.cloneNode(true);
+  // proper bg
+  copy.style.backgroundColor = 'white';
+  // inject font
+  copy.style.fontFamily = 'Barlow Condensed, sans-serif';
+  copy.insertAdjacentHTML('afterbegin', `<style>
+  @import url("https://fonts.googleapis.com/css?family=Barlow+Condensed");
+  </style>`);
```

Comment:
Does attempting to import the font have any effect? Seems it will just revert to sans-serif.

Reply:
when you switch to downloading the SVG image, you can see the font